core/src/avm2/globals/Date.as

```diff
@@ -1,5 +1,461 @@
-// This is a stub - the actual class is defined in `date.rs`
 package {
-    public class Date {
+    [Ruffle(InstanceAllocator)]
+    [Ruffle(CallHandler)]
+    public dynamic class Date {
+        public static const length:int = 7;
+
+        public function Date(year:* = undefined, month:* = undefined, day:* = undefined, hours:* = undefined, minutes:* = undefined, seconds:* = undefined, ms:* = undefined) {
+            this.init(arguments);
+        }
+        private native function init(args:Array);
+
+        public static native function parse(param1:*):Number;
+
+        public static native function UTC(param1:*, param2:*, param3:* = 1, param4:* = 0, param5:* = 0, param6:* = 0, param7:* = 0, ... rest):Number;
```

Comment:
We usually try to keep the naming from docs/playerglobal. These names do show up in exception messages (though currently not sure if this can happen in Ruffle, better have correct names just in case)

Reply:
I think this is the name in the playerglobal.